mirrord-agent/src/env.rs

```diff
@@ -0,0 +1,150 @@
+use std::{
+    collections::{HashMap, HashSet},
+    path::PathBuf,
+};
+
+use mirrord_protocol::RemoteResult;
+use tokio::io::AsyncReadExt;
+use tracing::trace;
+use wildmatch::WildMatch;
+
+struct EnvFilter {
+    include: Vec<WildMatch>,
+    exclude: Vec<WildMatch>,
+}
+
+impl EnvFilter {
+    fn new(select_env_vars: HashSet<String>, filter_env_vars: HashSet<String>) -> Self {
+        let include = if select_env_vars.is_empty() {
+            vec![WildMatch::new("*")]
+        } else {
+            select_env_vars
+                .iter()
+                .map(|selector| WildMatch::new(selector))
+                .collect()
+        };
+
+        let exclude = {
+            let mut exclude = Vec::new();
+
+            if !select_env_vars.contains("PATH") {
```

Comment:
Lets drop this code, for now. I doubt we'll ever have anyone actually use it, and if we do, I'd like to know